src/program/services/scrapers/jackett.py

```
@@ -110,52 +111,63 @@ def scrape(self, item: MediaItem) -> Dict[str, str]:
+                        for result, title in urls_to_fetch
+                    }
+
+                    for future in concurrent.futures.as_completed(future_to_result):
+                        result, title = future_to_result[future]
+                        try:
+                            infohash = future.result(timeout=30)
+                            if infohash:
+                                torrents[infohash] = title
+                        except concurrent.futures.TimeoutError:
+                            logger.debug(f"Timeout getting infohash from Link for {title}")
+                        except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)